config/settings/base.py

```diff
@@ -114,9 +114,12 @@
 # https://docs.djangoproject.com/en/dev/ref/settings/#auth-password-validators
 AUTH_PASSWORD_VALIDATORS = [
     {
-        "NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator"
+        "NAME": "django.contrib.auth.password_validation.UserAttributeSimilarityValidator",
+        "OPTIONS": {"user_attributes": ("username", "email", "name")},
```

Comment:
name is not valid here as some users can have similar names

Reply:
It checks the similarity between a user's password and its name only, not all names of the already existing users.

Reply:
okay make sense, thanks a lot